src/config.c

```diff
@@ -2729,7 +2720,7 @@ void rewriteConfigLatencyTrackingInfoPercentilesOutputOption(typeData data, cons
     rewriteConfigRewriteLine(state,name,line,1);
 }
 
-standardConfig configs[] = {
+standardConfig static_configs[] = {
```

Comment:
If we do want to move to this approach, we probably also want to replace initializing this array with just a series of functions to add the configs to the dict directly.

Reply:
I thought about doing some other refactoring, but I'll leave that to a separate PR as well.

Reply:
it'd rather avoid doing changes to the actual lines that declare each config.
this will cause merge conflicts for cherry picks, and merges with other branches and forks, as well as blame log destruction.